gp-nested-forms/gpnf-override-parent-merge-tag-on-submission.php

```diff
@@ -117,6 +117,10 @@ function override_child_entry_input_value( $entry, $field, $input_id, $default_v
 
 		$child_entry_ids = explode( ',', rgar( $entry, $field->id ) );
 		foreach ( $child_entry_ids as $child_entry_id ) {
+			// If any child entry merge tag is present, replace it with the child entry value before replacing with the Parent merge tag values.
+			$child_entry = GFAPI::get_entry( $child_entry_id );
+			$child_form  = GFAPI::get_form( $child_entry['form_id'] );
```

Comment:
Ideally `$child_form` shouldn't be in this foreach. It looks like the child form is retrieved on line 73 in `override_parent_merge_tags`.

I'd pass it to `override_child_entry_input_value` as another parameter.